Source/WebCore/svg/SVGAngleValue.h

```diff
@@ -34,7 +34,8 @@ class SVGAngleValue {
         SVG_ANGLETYPE_UNSPECIFIED = 1,
         SVG_ANGLETYPE_DEG = 2,
         SVG_ANGLETYPE_RAD = 3,
-        SVG_ANGLETYPE_GRAD = 4
+        SVG_ANGLETYPE_GRAD = 4,
+        SVG_ANGLETYPE_TURN = 5
```

Comment:
According to the style error, it seems I need to change the casing style of the enum type. 

```
UnspecifiedSvgAngleType = 1,
DegSvgAngleType = 2,
RadSvgAngleType = 3,
GradSvgAngleType = 4,
TurnSvgAngleType = 5
```

something like this ?

Reply:
If I were to clean up this code separately, I would switch to:
```
enum class Type : uint8_t {
  Unknown,
  Deg,
  Rad,
  Grad,
  Turn,
}
```

Definitely a separate task though.

Reply:
I added a FIXME in the source code.

Reply:
Should we add this bug link - https://bugs.webkit.org/show_bug.cgi?id=269429

Reply:
@nt1m How would you solve the difference with IDL?

https://searchfox.org/wubkat/source/Source/WebCore/svg/SVGAngleValue.h
https://searchfox.org/wubkat/source/Source/WebCore/svg/SVGAngle.idl

@Ahmad-S792 Yes!

Reply:
@Ahmad-S792 That said I added a pointer to the discussion here.

Reply:
Should we raise this issue to update IDL in SVGWG GitHub?

Reply:
This would be harder to change, given that this is implemented and available on the Web.